docs/api/linear_tutorial.rst

```diff
@@ -0,0 +1,95 @@
+Linear Classification Quickstart Tutorial
+=========================================
+
+.. role:: py(code)
+   :language: python3
+
+This guide will take you through how LibMultiLabel can
+be used to train a linear classifier in python scripts.
+
+Getting the Dataset
+^^^^^^^^^^^^^^^^^^^
+
+For this guide, we will use the rcv1 dataset, which is
+a collection of news articles.
+On the command line::
+
+    mkdir -p data/rcv1
+    wget https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/multilabel/rcv1_topics_train.txt.bz2 -O data/rcv1/train.txt.bz2
+    wget https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/multilabel/rcv1_topics_test.txt.bz2 -O data/rcv1/test.txt.bz2
+    bzip2 -d data/rcv1/*.bz2
+
+Each line of the dataset contains an ID, its labels and the
+text, all seperated by tabs. For example::
+
+    2286    E11 ECAT M11 M12 MCAT   recov recov recov recov excit excit bring mexic ...
+
+See `data formats <placeholder>`_ for a complete explanation of the format.
+
+The Python Script
+^^^^^^^^^^^^^^^^^
+
+For this guide, we will only need the linear module:
+
+.. literalinclude:: ../examples/linear_quickstart.py
+    :language: python3
+    :lines: 1
+
+To start, we need to read and preprocess the input data:
+
+.. literalinclude:: ../examples/linear_quickstart.py
+    :language: python3
+    :lines: 3-6
+
+The preprocessor handles many issues such as: mapping
+the labels into indices and transforming textual data to
+numerical data.
+
+Next we train the model:
+
+.. literalinclude:: ../examples/linear_quickstart.py
+    :language: python3
+    :lines: 8-10
+
+Where the third argument is the options string for
+`LibLinear <https://www.csie.ntu.edu.tw/~cjlin/liblinear/>`_.
+We may leave it as the default for now.
+
+Once we have the model, we may predict with it:
+
+.. literalinclude:: ../examples/linear_quickstart.py
+    :language: python3
+    :lines: 12
+
+Where :py:`preds` holds the decision values, i.e. the raw values
+outputted by the model. To transform it into predictions,
+the simplest way is to take the positive values as positive
+predictions, i.e. :py:`preds > 0`.
```

Comment:
this sentence doesn't read well. "positive predictions" not clear. rewrite it a bit